package.json

```
@@ -2,15 +2,25 @@
   "name": "yargs-parser",
   "version": "18.1.3",
   "description": "the mighty option parser used by yargs",
-  "main": "build/index.js",
+  "main": "build/index.cjs",
+  "exports": {
+    "import": "./build/index-node.js",
+    "require": "./build/index.cjs"
+  },
+  "type": "module",
+  "types": "./build/index.cjs.d.ts",
   "scripts": {
-    "fix": "standardx --fix ./*.ts && standardx --fix **/*.ts",
-    "pretest": "npm run compile -- -p tsconfig.test.json",
+    "check": "standardx && standardx ./*.ts && standardx **/*.ts && standardx test/deno/yargs-test.ts",
+    "fix": "standardx --fix && standardx --fix ./*.ts && standardx --fix **/*.ts && standardx --fix test/deno/yargs-test.ts",
+    "pretest": "rimraf build && tsc -p tsconfig.test.json && npm run build:cjs",
     "test": "c8 --reporter=text --reporter=html mocha test/*.js",
     "posttest": "npm run check",
     "coverage": "c8 report --check-coverage",
-    "check": "standardx ./*.ts && standardx **/*.ts",
-    "compile": "rimraf build && tsc",
+    "precompile": "rimraf build",
+    "compile": "tsc",
+    "postcompile": "npm run build:cjs",
+    "build:cjs": "rollup --exports=default --config=rollup.config.js ./index-node.ts --format cjs --file build/index.cjs",
```

A lot of these arguments could be configured more expressively in `./rollup.config.js`?
```suggestion
    "build:cjs": "rollup -c",
```
```js
export default {
  input: './index-node.ts',
  output: {
    format: 'cjs',
    file: './build/index.cjs',
    exports: 'default',
  },
  plugins: [
    ts({ /* options */ })
  ],
}
```